rsts/concepts/versioning.rst

```diff
@@ -3,47 +3,45 @@
 Versions
 ========
 
-One of the most important features and reasons for certain design decisions in Flyte is the need for machine learning and data practitioners to experiment.
-When users experiment, they usually work in isolation and try multiple iterations.
+One of the most important features and reasons for design decisions in Flyte is the need for machine learning and data practitioners to experiment.
+When users experiment, they do so in isolation and try multiple iterations.
 Unlike traditional software, the users must conduct multiple experiments concurrently with different environments, algorithms, etc.
-This may happen when different data scientists simultaneously iterate on the same workflow/pipeline.
+This may happen when multiple data scientists simultaneously iterate on the same workflow/pipeline.
 
-The cost of creating an independent infrastructure for each version is enormous and not desirable.
-Moreover, it is desirable to share the same centralized infrastructure, where the burden of maintaining the infrastructure is with a central infrastructure team,
-while users can use it independently. This also improves the cost of operation, since it is possible to reuse the same infrastructure for multiple teams.
+The cost of creating an independent infrastructure for each version is enormous but undesirable.
+It is beneficial to share the same centralized infrastructure, where the burden of maintaining the infrastructure is with a central infrastructure team,
+whereas the users can use it independently. This improves the cost of operation, since the same infrastructure can be reused by multiple teams.
```

Comment:
```suggestion
while the users can use it independently. This improves the cost of operation since the same infrastructure can be reused by multiple teams.
```